packages/node_modules/@webex/plugin-meetings/src/reconnection-manager/index.js

```diff
@@ -136,99 +139,114 @@ export default class ReconnectionManager {
   }
 
   /**
-   * @param {Meeting} meeting
    * @returns {Boolean}
    * @returns {ReconnectionError}
    * @private
    * @memberof ReconnectionManager
    */
-  validate(meeting) {
-    // TODO: re write the validate function
-    if (meeting.config.reconnection.enabled) {
+  validate() {
```

Comment:
API breaking, but only seems to be referenced by `this` object.

Reply:
validate is listed as a private function :)